src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -132,5 +134,11 @@ object Instance {
   implicit val idFormat = Json.format[Instance.Id]
   implicit val instanceStatusFormat = Json.format[InstanceStatus]
   implicit val instanceStateFormat = Json.format[InstanceState]
-  implicit val instanceJsonFormat = Json.format[Instance]
+  implicit val instanceJsonFormat: Format[Instance] = Json.format[Instance]
+
+  import mesosphere.mesos.Placed
+  implicit class PlacedInstance(instance: Instance) extends Placed {
```

Comment:
Do you have a suggested alternative?

On Wed, Sep 14, 2016 at 12:15 PM, Jason Gilanfarr <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/core/instance/Instance.scala
> <https://github.com/mesosphere/marathon/pull/4382#discussion_r78781874>:
>
> > @@ -132,5 +134,11 @@ object Instance {
> >    implicit val idFormat = Json.format[Instance.Id]
> >    implicit val instanceStatusFormat = Json.format[InstanceStatus]
> >    implicit val instanceStateFormat = Json.format[InstanceState]
> > -  implicit val instanceJsonFormat = Json.format[Instance]
> > +  implicit val instanceJsonFormat: Format[Instance] = Json.format[Instance]
> > +
> > +  import mesosphere.mesos.Placed
> > +  implicit class PlacedInstance(instance: Instance) extends Placed {
>
> I'm not personally a fan.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4382/files/5081cb5061b0a7173ed46219bc9f0bef2b1f58d7#r78781874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLLa6h92cenz9ffHcYBXZuzbRxJlHks5qqB2fgaJpZM4J8mwh>
> .
>